### PR TITLE
fix(bug): fix discord link

### LINKED ISSUE #217: Freecodecamp Discord Link Invalid
### Describe the Issue

The link for discord is invalid: https://discord.com/invite/freecodecamp-org-official-fi-fo-692816967895220344.

### Affected Page

https://discord.com/invite/freecodecamp-org-official-fi-fo-692816967895220344

### Steps to Reproduce

Click on the discord link in the upper right corner.

### Recommended fix or suggestions

Update the existing link for discord.

### Screenshots

Screenshot from 2024-07-14 20-47-14

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #217

### CODE REVIEW

**Comment by michaelhiiva** [comment]
@gagan-bhullar-tech The link still does not work for me. I think you are looking for https://discord.com/invite/freecodecamp which is on https://discord.com/servers/freecodecamp-org-official-fi-fo-692816967895220344.

**Comment by gagan-bhullar-tech** [comment]
@michaelhiiva thanks. Link is updated.

**Comment by Blindspot22** [comment]
> @michaelhiiva thanks. Link is updated.

@gagan-bhullar-tech Seems like this your PR has not still yet been approved for merging.
Maybe you should try tagging the reviewer in the comment section again for it to be reviewed.

File: astro.config.mjs [hunk 1]
```diff
@@ -63,8 +63,7 @@ export default defineConfig({
       social: {
         github: 'https://github.com/freeCodeCamp',
         twitter: 'https://twitter.com/freeCodeCamp',
-        discord:
-          'https://discord.com/invite/freecodecamp-org-official-fi-fo-692816967895220344'
+        discord: 'https://discord.com/invite/freecodecamp'
```
Comment (raisedadead) [inline astro.config.mjs]:
```suggestion
        discord: 'https://chat.freecodecamp.org'
```

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
Thanks for the PR - I have updated it after discussing it with the team. We are going to use a more permanent public link instead.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**